### ISSUE TITLE
JetBrains Mono NL, some symbol (e.g. , Apple icon) missing after patched.

### ISSUE THREAD

**Issue by nyngwang** [ryanoasis/nerd-fonts#603]
#### 🎯 Subject of the issue

TL;DR: 
![image](https://user-images.githubusercontent.com/24765272/113388282-85a32e80-93c0-11eb-9e77-7de66ee49366.png)
becomes 
![image](https://user-images.githubusercontent.com/24765272/113388014-fc8bf780-93bf-11eb-9b15-4821228e24e1.png)

#### 🔧 Your Setup

- _Which font are you using (e.g. `Anonymice Powerline Nerd Font Complete.ttf`)?_

I've patched Nerd Font to both (1)Entire `JetBrains Mono NL` font family downloaded from JetBrains' offical site, and (2) Downloaded the patched JetBrainsMono NL Complete `.ttf` from this repository.

Both the same result.

- _Which terminal emulator are you using (e.g. `iterm2`, `urxvt`, `gnome`, `konsole`)?_

Terminal and Alacritty.

Both the same result.

- _Are you using OS X, Linux or Windows? And which specific version or distribution?_

MacOSX Big Sur ver11.2.3 (20D91)

#### ★ Optional

Without Nerd Font:
![image](https://user-images.githubusercontent.com/24765272/113387899-c64e7800-93bf-11eb-90e8-8be44182a0f2.png)

With Nerd Font:
![image](https://user-images.githubusercontent.com/24765272/113387940-dbc3a200-93bf-11eb-913e-bb3bc0c0e98d.png)

**Comment by gubler** [comment]
Pretty sure this is the same problem as #492

**Comment by ryanoasis** [comment]
`F8FF` resides in the Private Use Area so it shouldn't be protected from modification (it's free game), that said this is coming from material design icons and Nerd Fonts character range for those is problematic and needs to be moved anyway.

**Comment by cgarrovillo** [comment]
so what exactly needs to happen now

**Comment by jukrb0x** [comment]
> Same issue. `FiraMono Nerd Font`, `FiraCode Nerd Font` missing the  `\uF8FF` symbol.
> 
> Is it a feature?

To whom really needs the  Apple symbol,

I am not sure what can this repository do. But recently I've changed to `power10k` for my zsh terminal and ref to [this document](https://github.com/romkatv/powerlevel10k#fonts), they have the patch for `Meslo Nerd Font` which the Apple  symbol works well on it. But the  `\uF8FF` still hold as symbol pi  ![image](https://user-images.githubusercontent.com/15688641/156300195-2c0aaee4-7671-4f3c-8ee4-f9f948f9f4d6.png), the symbol Apple is located at  `\uf179` as the screenshot below. This symbol looks like strips in my markdown editor ![image](https://user-images.githubusercontent.com/15688641/156300650-24547f11-40a1-4412-95ed-4335216afe8e.png), Apple symbol also occurred in some [other practice](https://ohmyposh.dev/docs/os) of this unicode.





![image](https://user-images.githubusercontent.com/15688641/156299917-8e58ef01-af34-4565-a0d8-c163a214f8e2.png)

**Comment by Finii** [comment]
Hmm, Nerd Font has the bitten off fruit on `u_F302`, why not just change the codepoint in your prompt?

![image](https://user-images.githubusercontent.com/16012374/156302336-44e3680c-e9ee-462f-8a66-7231ffc194fb.png)

**Comment by cgarrovillo** [comment]
I have the patched Meslo nerd font, but even then:

<img width="316" alt="image" src="https://user-images.githubusercontent.com/6688713/156409273-9ebab756-192c-452a-b06d-702419a16590.png">

its the weirdest thing

**Comment by Finii** [comment]
Obviously what you entered for the prompt is escaped and/or not expanded. Try copy and pasting that one character directly into the prompt-defining file, not using the sequence to define it.

**Comment by cgarrovillo** [comment]
Lol i copy and pasted it before but with a different unicode. Figured pasting the character directly doesn't work. 

Thanks

**Comment by Finii** [comment]
Problem with `F8FF` is fixed in 3.0.0
